yubikit/README.md

```diff
@@ -77,9 +78,11 @@ and/or
             // NFC is not available so this feature doesn't work on this device
         }
     }
-```
--4. Open ISO/IEC 7816 connection from YubiKey session (NfcSession or UsbSession), check ATR, create APDU and execute it. Note: the API that send APDU commands to YubiKey are blocking and require background thread to be created in order to provide good user experience.
-```java
+    ```
+6. Open an ISO/IEC 7816 connection from YubiKey session (NfcSession or UsbSession), check ATR, create APDU, and then execute it. 
```

Comment:
we should mention here that user has option to use our high-level APIs for different yubiKey application (and may be point back to the list of modules that we support?)